### ISSUE TITLE
Integrations/Sync with Tag Filter

### ISSUE THREAD

**Issue by rawkode** [Infisical/infisical#1348]
### Feature description
As I continue to explore Infisical for mono-repo setups; I would like to configure an integration to Supabase and GitHub; but I don't want to sync all the secrets within a folder.

Instead, I'd like to sync all the secrets within a folder (or project) that are tagged with a certain tag.

Related: #243

### Why would it be useful?
I've tried to do this on a folder level, but it requires lots of secret references which aren't ideal as they don't work well with a mono repository hierarchy and all references need to be absolute as `../` or `folder_from_here/` don't work.

### Additional context
N/A

**Comment by akhilmhdh** [comment]
@rawkode have you tried secret import feature. https://infisical.com/docs/documentation/platform/secret-reference#import-entire-folders

This will pull all the secrets of any environment or any path that you have access to

**Comment by rawkode** [comment]
@akhilmhdh No, though that will definitely help minimise the toil. I could have a special "integrations" folder with subfolders for syncs; then pull them into sub-folders, but it will still be a bit more effort than I'd like.

Being able to say "Sync all Supabase tagged secrets to Supabase" is a much more flexible solution, especially if it can traverse folders.

Let me know if this is a feature that would be welcomed and I'll work on a PR

**Comment by akhilmhdh** [comment]
So if my understanding is right. 

You want to also have an additional filter by tag while integration syncing. 

Hmm we are soon gonna do an integration revamp. 

CC: @dangtony98 (for visibility)

**Comment by rawkode** [comment]
🙁

This is disappointing. I'd have been happy to contribute this myself
